pages/search/formatters/index.jsx

```diff
@@ -11,12 +11,15 @@ import format from 'date-fns/format';
 
 const DATE_FORMAT = 'D MMM YYYY';
 
+// markdown bolding breaks if there's a space before the closing markers
+const markdownSafe = highlight => highlight && highlight.replace(/ \*\*/g, '** ');
```

Comment:
I'm pretty sure it needs to be _a lot_ cleverer than this.

e.g. `When highlighting **this** sentence it's going to break.`